Objects/funcobject.c

```diff
@@ -690,8 +689,64 @@ _Py_set_function_type_params(PyThreadState *Py_UNUSED(ignored), PyObject *func,
     return Py_NewRef(func);
 }
 
+static PyObject *
+func_get_closure(PyFunctionObject *op, void *Py_UNUSED(ignored))
+{
+    if (op->func_closure == NULL) {
+        Py_INCREF(Py_None);
```

Comment:
`Py_None` is immortal now, but I don't know what the policy is about its use in `Objects/`.

I think that people still use it.